sdk/bare/common/drv/eddy_current_sensor.c

```diff
@@ -40,26 +42,39 @@ void eddy_current_sensor_set_divider(uint8_t divider)
     Xil_Out32(EDDY_CURRENT_SENSOR_BASE_ADDR + (2 * sizeof(uint32_t)), divider);
 }
 
+double eddy_current_sensor_bits_to_voltage(uint32_t data)
```

Comment:
Since this is a private function not defined in the header file, please change to:

```C
static double bits_to_volts(uint32_t data)
```